BraveRewardsUI/Settings/Auto-Contribute/Details/AutoContributeDetailsViewController.swift

```diff
@@ -273,40 +286,56 @@ extension AutoContributeDetailViewController: UITableViewDataSource, UITableView
       case .monthlyPayment:
         cell.label.text = Strings.AutoContributeMonthlyPayment
         cell.accessoryType = .disclosureIndicator
-        if let walletInfo = ledger.walletInfo, let dollarAmount = ledger.dollarStringForBATAmount(ledger.contributionAmount) {
-          cell.accessoryLabel?.text = "\(ledger.contributionAmount) \(walletInfo.altcurrency) (\(dollarAmount))"
+        if let walletInfo = state.ledger.walletInfo, let dollarAmount = state.ledger.dollarStringForBATAmount(state.ledger.contributionAmount) {
+          cell.accessoryLabel?.text = "\(state.ledger.contributionAmount) \(walletInfo.altcurrency) (\(dollarAmount))"
         }
       case .nextContribution:
         cell.label.text = Strings.AutoContributeNextDate
         cell.accessoryView = nextContributionDateView
         cell.selectionStyle = .none
       case .supportedSites:
         cell.label.text = Strings.AutoContributeSupportedSites
-        cell.accessoryLabel?.attributedText = totalSitesAttributedString(from: upcomingContributions.count)
+        cell.accessoryLabel?.attributedText = totalSitesAttributedString(from: publishers.count)
         cell.selectionStyle = .none
       case .excludedSites:
-        // FIXME: Use actual number
-        let numberOfExcludedSites = 5
+        let numberOfExcludedSites = state.ledger.numberOfExcludedPublishers
         cell.label.text = String(format: Strings.AutoContributeRestoreExcludedSites, numberOfExcludedSites)
         cell.label.textColor = Colors.blurple400
       }
       return cell
     case .contributions:
-      if upcomingContributions.isEmpty {
+      if publishers.isEmpty {
         let cell = tableView.dequeueReusableCell(for: indexPath) as EmptyTableCell
         cell.label.text = Strings.EmptyAutoContribution
         return cell
       }
-      let contribution = upcomingContributions[indexPath.row]
+      guard let publisher = publishers[safe: indexPath.row] else {
+        assertionFailure("No Publisher found at index: \(indexPath.row)")
+        return UITableViewCell()
+      }
       let cell = tableView.dequeueReusableCell(for: indexPath) as AutoContributeCell
       cell.selectionStyle = .none
       cell.siteImageView.image = UIImage(frameworkResourceNamed: "defaultFavicon")
-      cell.verifiedStatusImageView.isHidden = !contribution.isVerified
-      cell.siteNameLabel.text = contribution.site
-      cell.attentionAmount = contribution.attention
+      cell.verifiedStatusImageView.isHidden = !publisher.verified
+      let provider = "\(publisher.provider.isEmpty ? "" : "on \(publisher.provider)")"
```

Comment:
Same thing as #101 I think

Let me know if design is different on AC vs tips table

Reply:
Nah its same, just waiting for the #101 merge so that I can pull the Strings.swift changes.